experiments/Schema/src/Testing.jl

```diff
@@ -1,15 +1,15 @@
 # here is an example
```

Comment:
should this file move to path `experiments/Schema/test/runtests.jl`?

Reply:
Good idea, I'll write it into a `Test.jl` testing suite for the package.